returnn/tf/layers/basic.py

```diff
@@ -852,46 +853,54 @@ class SliceNdLayer(_ConcatInputLayer):
   layer_class = "slice_nd"
   recurrent = True
 
-  def __init__(self, start, size, min_size=None, **kwargs):
+  def __init__(self, start, size, **kwargs):
     """
-    :param LayerBase start:
-    :param int|None size: if None, it uses the max possible size, and it becomes a dynamic axis
-    :param int|None min_size: if size is None, but we want to have a min-size, set this
```

Comment:
You are completely changing the functionality of the layer. You remove a lot of functionality which previously existed. E.g. the `min_size` option. Or that `size` can be `None`.

We do not allow this (unless you make sure that all existing RETURNN setups from everyone do not make use of this, and also any important previous setup, or e.g. anything on returnn-experiments, etc, which is pretty hard to do).

Reply:
Oh yes, I didn't think about that. I will change that